src/screens/Ticketing/Purchase/TariffZones/search/TariffZoneResults.tsx

```diff
@@ -10,7 +10,7 @@ import {TariffZoneSearchTexts, useTranslation} from '@atb/translations';
 import insets from '@atb/utils/insets';
 import React from 'react';
 import {ScrollView, TouchableOpacity, View} from 'react-native';
-import {useTariffZoneFromLocation} from '@atb/screens/Ticketing/Purchase/Overview';
+import {useTariffZoneFromLocation} from '../../utils';
```

Comment:
Think this should be absolute with `@atb...`

Reply:
Not sure what the logic is but, VSCode suggests `../../utils` here by default. Do we want to manually update the defaults in cases like these? It's slightly more work if we can't change default settings, but I agree it's more readable with `@atb...`.